### PR TITLE
DOC: Update to release upcoming change document

### PR DESCRIPTION
**Pull request by salujaditi14**

### CODE REVIEW

File: doc/release/upcoming_changes/23762.new_function.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Matrix transpose support for ndarrays
 -------------------------------------
 NumPy now offers support for calculating the matrix transpose of an array. The
-matrix transpose is equivalent to swapping the last two axes of an array.
-`np.ndarray` and `np.ma.MaskedArray` now exposes a `.mT` attribute.
\ No newline at end of file
+matrix transpose is equivalent to swapping the last two axes of an array. The
```
Comment (charris) [inline doc/release/upcoming_changes/23762.new_function.rst:4]: I'd leave out the new "the" and say "Both `np.narray` and ...".  ".mT" and the rest should be in double backticks.

Comment (salujaditi14) [reply doc/release/upcoming_changes/23762.new_function.rst]: Fixed in new commit

File: doc/release/upcoming_changes/23762.new_function.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Matrix transpose support for ndarrays
 -------------------------------------
 NumPy now offers support for calculating the matrix transpose of an array. The
-matrix transpose is equivalent to swapping the last two axes of an array. The
-`np.ndarray` and the `np.ma.MaskedArray` now expose a `.mT` attribute.
+matrix transpose is equivalent to swapping the last two axes of an array. Both
+``np.ndarray`` and ``np.ma.MaskedArray`` now expose an ``.mT`` attribute.
```
Comment (salujaditi14) [inline doc/release/upcoming_changes/23762.new_function.rst:5]: @charris Updated the comments

**Comment by charris** [comment]
Thanks @salujaditi14 .

**Pull request merged**